kingfisher_scrapy/spiders/mexico_plataforma_digital_nacional.py

```diff
@@ -0,0 +1,31 @@
+import scrapy
+
+from kingfisher_scrapy.base_spider import CompressedFileSpider
+from kingfisher_scrapy.util import components, handle_http_error
+
+
+class MexicoPlataformaDigitalNacional(CompressedFileSpider):
+
+    """
+    Domain
+      Plataforma Digital Nacional
+    Bulk download documentation
+      https://plataformadigitalnacional.org/contrataciones
+    """
+    name = 'mexico_plataforma_digital_nacional'
+    data_type = 'release'
+    root_path = 'item'
```

Comment:
All spiders currently follow this format, to make it easier to figure out where the behavior associated to the class attribute can be found.

```suggestion

    # BaseSpider
    root_path = 'item'

    # CompressedFileSpider
    data_type = 'release'
```

Reply:
right! I guess it has been so long since the last time I wrote a spider from scratch that I forgot this, thanks!

Reply:
No worries :)